index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
```suggestion
ghfghfg}]asd]ad]as[dasd]
```
```
```
```
```
```
```
```
```
```
```